frontend/test/pytest/test_config_functions.py

```diff
@@ -33,6 +33,13 @@
 from catalyst.utils.toml import toml_load
 
 
+class TestDevice(qml.QubitDevice):
```

Comment:
not related to this PR -- but I'm wondering if there are other places that we can use this TestDevice in frontend tests :thinking:

Reply:
I think we use such devices in other tests here and there as well :)